src/api/axiosConfig.js

```diff
@@ -1,8 +1,13 @@
 import axios from "axios";
 
-const instance = axios.create({
+const apiDev = axios.create({
```

Comment:
Why was this renamed? also have you consulted the others on this since everyone is still using instance on their work?

Reply:
We are exporting the instance as default. My assumption is that when we export it as a default, the name does not matter when importing.

Reply:
👍